src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFMerger.java

```diff
@@ -82,24 +88,33 @@ private void addFolderCoversheet(SortableBundleItem item) throws IOException {
                 addText(document, page, item.getDescription(), 50, 80, PDType1Font.HELVETICA,12);
             }
 
+            pdfOutline.addFolder(currentPageNumber, item.getTitle());
             currentPageNumber++;
         }
 
         private void addDocument(SortableBundleItem item) throws IOException {
             PDDocument newDoc = PDDocument.load(documents.get(item));
+
+            PDDocumentOutline newDocOutline = newDoc.getDocumentCatalog().getDocumentOutline();
+            newDoc.getDocumentCatalog().setDocumentOutline(null);
+
             merger.appendDocument(document, newDoc);
 
             if (bundle.getPaginationStyle() != PaginationStyle.off) {
                 addPageNumbers(document, bundle.getPaginationStyle(), currentPageNumber, currentPageNumber + newDoc.getNumberOfPages());
             }
 
-            newDoc.close();
-
             if (tableOfContents != null) {
                 tableOfContents.addDocument(item.getTitle(), currentPageNumber, newDoc.getNumberOfPages());
             }
 
+            pdfOutline.addFolder(currentPageNumber, item.getTitle());
+            if (newDocOutline != null) {
+                pdfOutline.mergeDocumentOutline(currentPageNumber, newDocOutline);
+            }
+            pdfOutline.closeFolder();
```

Comment:
How come we have this here and also in the `addContainer` method?

Reply:
i was talking to jamil on whether the original outline should be a child or sibling, did get an answer and it will be a sibling. its in addContainer as we need it when adding folders

Reply:
So shouldn't closeFolder be inside the if statement then?

Reply:
i got rid of it in addDocument as discussing it with jamil now, he wanted it as a sibling instead of child. so no need to for it now